runtime-sdk/src/module.rs

```diff
@@ -277,7 +277,7 @@ pub trait AuthHandler {
         _tx: &Transaction,
     ) -> Result<(), modules::core::Error> {
         // Default implementation doesn't do any checks.
```

Comment:
Change this comment to say that the default implementation rejects all transactions.